WordPress/UITestsFoundation/Screens/TabNavComponent.swift

```diff
@@ -71,6 +71,7 @@ public class TabNavComponent: ScreenObject {
 
     public func goToNotificationsScreen() throws -> NotificationsScreen {
         notificationsTabButton.tap()
+        try dismissNotificationAlertIfNeeded()
```

Comment:
Tests were failing because there was an extra notification that needed to be dismissed. Adding this seems to fix it, now tests are passing locally and in CI.